### PR TITLE
Add english translations

### PR DESCRIPTION
**Pull request by tonyyb**

### CODE REVIEW

**Review by Seb-C — APPROVED** [review]

**Comment by DOMEYD** [comment]
:+1:

File: lang/en/default.lang.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+return array(
+    'Your Google SEO Tools' => 'Your Google SEO Tools',
+    'Paramètres Google Analytics' => 'Google Analytics parameters',
+    'Google webmaster tools' => 'Google webmaster tools',
+    'Paramètres expert' => 'Expert parameter',
+    'Tag google analytics' => 'Google Analytics tag',
+    'Script entier' => 'Full script',
+    'Méthode de vérification du site' => 'Website verification method',
+    'Contenu de la balise meta' => 'Meta tag content',
+    'OU (utiliser les deux méthodes est inutile)' => 'OR (use the 2 methods is useless)',
```
Comment (Meroje) [inline lang/en/default.lang.php]: s/use/using/

File: lang/en/default.lang.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    'Nom de la page html de vérification' => 'Name of HTML verification page',
```
Comment (Meroje) [inline lang/en/default.lang.php]: Name of **the**

File: lang/en/default.lang.php [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    'Utiliser Universal analytics' => 'Use Universal analytics',
+    'Cookie de tracking' => 'Tracking cookie',
+    'Nom du cookie' => 'Name of cookie',
```
Comment (Meroje) [inline lang/en/default.lang.php]: Cookie name

File: lang/en/default.lang.php [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    'Valeur du cookie' => 'Value of cookie',
```
Comment (Meroje) [inline lang/en/default.lang.php]: Cookie value

File: lang/en/default.lang.php [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+return array(
+    'Your Google SEO Tools' => 'Your Google SEO Tools',
+    'Paramètres Google Analytics' => 'Google Analytics parameters',
```
Comment (Meroje) [inline lang/en/default.lang.php]: I would use "settings" in place of "parameters"

**Review by Meroje — CHANGES_REQUESTED** [review]

File: lang/en/default.lang.php [hunk 6]
```diff
@@ -2,19 +2,19 @@
...
+    'Paramètres Google Analytics' => 'Google Analytics settings',
     'Google webmaster tools' => 'Google webmaster tools',
     'Paramètres expert' => 'Expert parameter',
     'Tag google analytics' => 'Google Analytics tag',
     'Script entier' => 'Full script',
     'Méthode de vérification du site' => 'Website verification method',
     'Contenu de la balise meta' => 'Meta tag content',
-    'OU (utiliser les deux méthodes est inutile)' => 'OR (use the 2 methods is useless)',
-    'Nom de la page html de vérification' => 'Name of HTML verification page',
+    'OU (utiliser les deux méthodes est inutile)' => 'OR (use these two methods is useless)',
```
Comment (Meroje) [inline lang/en/default.lang.php]: using

**Review by Meroje — CHANGES_REQUESTED** [review]

**Review by Meroje — APPROVED** [review]

**Pull request merged**